contract/src/main/java/io/neow3j/contract/NeoURI.java

```diff
@@ -45,8 +45,8 @@ public NeoURI(Neow3j neow3j) {
      * @throws IllegalFormatException if the provided URI has an invalid format.
      */
     public static NeoURI fromURI(String uriString) throws IllegalFormatException {
-        String[] base_and_query = uriString.split("\\?");
-        String[] beginTx = base_and_query[0].split(":");
+        String[] baseAndQuery = uriString.split("\\?");
+        String[] beginTx = baseAndQuery[0].split(":");
```

Comment:
Is there any chance that `baseAndQuery` does not have a `0` index position?

Reply:
No, even if the uriString is an empty String or if the first entry of the uriString is the regex, the `.split()` will add an empty String to the index `0`. So such a uriString is then handled in the `if` condition below and a IllegalArgumentException is thrown.